### PR TITLE
A futher round of noskips

### PR DESCRIPTION
**Pull request by psav**
A futher round of noskips

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit fa8356b79c159f20de214956dd7729de4fac3d0f:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/control/test_actions.py [hunk 1]
```diff
@@ -49,11 +50,27 @@ def __getattr__(self, key):
...
+    new_idlist = []
+    new_argvalues = []
+    for i, argvalue_tuple in enumerate(argvalues):
+        args = dict(zip(argnames, argvalue_tuple))
+
+        if ((metafunc.function is test_action_create_snapshot_and_delete_last)
+            or
+            (metafunc.function is test_action_create_snapshots_and_delete_them)) \
+                and isinstance(args['provider_crud'], RHEVMProvider):
+            continue
```
Comment (mfalesni) [inline cfme/tests/control/test_actions.py]: I wonder if we could make some kind of uncollect mark that would enable us to not write custom generate_tests hooks ...

Comment (psav) [inline cfme/tests/control/test_actions.py]: Yes this is in the works....

**Comment by psav** [comment]
Failures are not related to collection

**Pull request merged**